### PR TITLE
Feat/cd pipeline

### PR DESCRIPTION
**Pull request by ninadarbaidze**

### CODE REVIEW

File: components/Movies/ImageEditInput/useImageEditInput.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { useTranslation } from 'next-i18next';
 import { MovieContext } from 'store';
 import { FieldProps } from 'formik';
 
-export const useImageEditInput = (props: { form: FieldProps }) => {
+export const useImageEditInput = (props: { form: FieldProps | any }) => {
```
Comment (GigaGiorgadze) [inline components/Movies/ImageEditInput/useImageEditInput.ts]: what did i say about anys >:( is it truly needed here? i don't think it really is if im wrong tell me

File: components/Movies/ImageEditInput/useImageEditInput.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { useTranslation } from 'next-i18next';
...
   const { form } = props;
   const [imagePreview, setImagePreview] = useState<any>();
```
Comment (GigaGiorgadze) [inline components/Movies/ImageEditInput/useImageEditInput.ts:11]: also if this is image preview and u are using createObject Url i think type of this is File,

File: components/icons/DeleteIcon.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { Click } from 'types';
+import { OnClick } from 'types';
 
-const DeleteIcon: React.FC<Click> = (props) => {
+const DeleteIcon: React.FC<OnClick> = (props) => {
   const { onClick } = props;
```
Comment (GigaGiorgadze) [inline components/icons/DeleteIcon.tsx]: doesn't make sense to have click on on SVG, it should be on button/div that is sits inside

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: components/Movies/ImageEditInput/useImageEditInput.ts (hunk 1 above)
Comment (ninadarbaidze) [reply components/Movies/ImageEditInput/useImageEditInput.ts]: image

I have problems with setFieldValue, spend hours on any yesterday and couldn't find solution for setFieldValue type

File: components/icons/DeleteIcon.tsx (hunk 3 above)
Comment (GigaGiorgadze) [reply components/icons/DeleteIcon.tsx]: its still here, remove it from icon

File: components/Movies/ImageInput/useImageInput.ts [hunk 4]
```diff
@@ -2,11 +2,11 @@ import { useCallback, useState } from 'react';
 import { useDropzone } from 'react-dropzone';
 
 import { useTranslation } from 'next-i18next';
-import { FieldProps } from 'formik';
+import { FormikHelpers } from 'formik';
 
-export const useImageInput = (props: { form: FieldProps }) => {
+export const useImageInput = (props: { form: FormikHelpers<File> }) => {
```
Comment (GigaGiorgadze) [inline components/Movies/ImageInput/useImageInput.ts]: look i think this is wrong but correct me if im wrong, look in formikHelpers u should pass type of a form something like this: { image: File | strign}  this is valid but the type u wrote would read like this FormikHelpers<last_edit_at: string; type: jpg> and along those lines, and u do not have those values in form do you? u should type fields name and its type

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

File: components/icons/DeleteIcon.tsx [hunk 5]
```diff
@@ -1,30 +1,31 @@
...
-          fill='#6C757D'
-        />
-      </g>
-      <defs>
-        <clipPath id='clip0_4351_16735'>
-          <rect width='14' height='14' fill='white' />
-        </clipPath>
-      </defs>
-    </svg>
+    <button onClick={onClick}>
```
Comment (GigaGiorgadze) [inline components/icons/DeleteIcon.tsx]: no... onClick should not be in icon move it out, icon are for icons,

**Review by GigaGiorgadze — CHANGES_REQUESTED** [review]

**Review by GigaGiorgadze — APPROVED** [review]

**Pull request merged**